vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -1377,7 +1379,7 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
                                         mSession.myUser.updateAvatarUrl(contentUri, object : ApiCallback<Void> {
                                             override fun onSuccess(info: Void?) {
                                                 onCommonDone(null)
-                                                refreshDisplay()
+                                                hideLoadingView(true)
```

Comment:
This is already done in `onCommonDone()` so I think this can be removed.

Reply:
onCommonDone() calls just hideLoadingView() doesn't refreshDisplay(). So I see , we can call refreshDisplay() only